.ci/run_tests.sh

```diff
@@ -82,10 +82,13 @@ if test -d ${QGIS_BUILDDIR} -a "${FORCE_REBUILD}" = "no"; then
   echo "--=[ Testing against pre-existing build directory ${QGIS_BUILDDIR}. To rebuild use --force-rebuild or move it away"
 else
   echo "--=[ Building qgis inside the dependencies container"
+  VOLUMES="-v ${QGIS_WORKSPACE}:${QGIS_WORKSPACE}"
```

Comment:
```suggestion
  VOLUMES="-v ${QGIS_WORKSPACE}:${QGIS_WORKSPACE}:z"
```

These would make it work with SELinux, but I guess nobody runs it.